rpc_jobs/rpc_maas.yml

```diff
@@ -1,3 +1,29 @@
+- project:
+    name: "rpc-maas-master-osp-mnaio-bionic-premerge"
```

Comment:
Is the intent to do a MNAIO build for all PR's, or was this actually intended for periodic jobs?

Reply:
It is actually for all PRs for now. Need to get in a PR to pass the pre-merge gate, then enable periodic release gate.